doc/source/user_guide/hpc_slurm.rst

```diff
@@ -0,0 +1,609 @@
+- Keep batch scripts and job submissions concise and well-documented 
+  for reproducibility and troubleshooting.
+
+.. _ref_hpc_troubleshooting:
+
+Troubleshooting
+===============
+
+Debugging Jobs
+--------------
+- Use ``--output`` and ``--error`` directives in batch scripts to capture
+  standard output and error messages. 
+
+- Check SLURM logs for error messages and debugging information.
+
+
+.. _ref_python_venv_not_accesible:
+
+Python virtual environment is not accessible
+--------------------------------------------
+If there is an error while testing the Python installation, it might mean 
+that the Python environment is not accessible to the compute nodes.
+For example, in the following output, PyMAPDL could not be found, meaning that the script
+is not using the virtual environment ``/home/user/.venv``:
+
+.. code-block:: console
+
+    user@machine:~$ srun test.sh
+    Testing Python!
+    Traceback (most recent call last):
+    File "<string>", line 1, in <module>
+    ImportError: No module named ansys.mapdl
+
+This could be for a number of reasons. One of them is that the system Python distribution
+used to create the virtual environment is not accessible from the compute nodes.
+Either because the virtual environment has been created in a directory which is not accessible
+from the nodes or because the virtual environment has been created from a Python executable
+which is not available to the compute nodes, hence the virtual environment is not activated.
+For example, you might be creating the virtual environment using Python 3.10, but only
+Python 3.8 is available from the compute nodes.
+
+You can test which Python executable the cluster is using by starting an interactive session in
+a compute node using:
+
+.. code-block:: console
+
+    user@machine:~$ srun --pty /bin/bash
+    user@compute_node_01:~$ compgen -c | grep python # List all commands starting with python
+
+.. the approach to solve this comes from:
+   https://stackoverflow.com/questions/64188693/problem-with-python-environment-and-slurm-srun-sbatch
+
+Many HPC infrastructure uses environment managers to load and unload software package using modules
+and environment variables. 
+Hence you might want to make sure that the correct module is loaded in your script.
+Two of the most common environment managers are
+`Modules documentation <modules_docs_>`_ and `Lmod documentation <lmod_docs_>`_.
+Check your cluster documentation to know which environment manager is using, and how to
+load Python with it. If you find any issue, you should contact your cluster administrator.
+
+If there is not a suitable Python version accessible from the compute nodes, you might need
+request to your HPC cluster administrator to have installed in all the compute
+nodes a suitable Python version.
+If this is not an option, visit :ref:`ref_ansys_provided_python`.
+
+.. _ref_ansys_provided_python:
+
+Using ANSYS provided Python installation
```

Comment:
Pinging @koubaa  ... shall we include this? I think we should ... stating it is for dev purposes only.

Reply:
I agree to include